test/e2e/create_staticip_test.go

```diff
@@ -74,7 +73,7 @@ var _ = Describe("Podman create with --ip flag", func() {
 		// test1 container is running with the given IP.
 		result = podmanTest.Podman([]string{"start", "-a", "test2"})
 		result.WaitWithDefaultTimeout()
-		Expect(result).To(ExitWithError())
+		Expect(result).To(ExitWithError(125))
```

Comment:
AFAIK we dropped CNI tests so could you just simply this block below and only match the netavark message?